src/txmempool.h

```diff
@@ -880,15 +880,28 @@ class CTxMemPool
  * It also allows you to sign a double-spend directly in
  * signrawtransactionwithkey and signrawtransactionwithwallet,
  * as long as the conflicting transaction is not yet confirmed.
+ * It can also serve as temporary scratch space for some set
```

Comment:
> Rather than extending CCoinsViewMemPool, would it make sense to instead add something like CCoinsViewPackage : CCoinsViewBacked?

This is a really interesting idea, and I think it should be the case to have the "top level" have the package caches -> "bottom level" backend is the `CCoinsViewMemPool`. Let me try to do it this way and see what happens!

Reply:
Note that `CCoinsViewMemPool` is created with `MemPoolAccept` instances so we don't really reuse an existing one.